Gemfile.lock

```diff
@@ -219,4 +219,4 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
 
```

Comment:
you can also remove geocoder, jquery-cookie-rails, mongoid-elasticsearch if you don't need it.